CMakeLists.txt

```diff
@@ -515,7 +520,7 @@ set_target_properties(torrent-rasterbar
 	PROPERTIES
 		CXX_VISIBILITY_PRESET "hidden"
 		VISIBILITY_INLINES_HIDDEN "true"
-		VERSION ${PROJECT_VERSION}
+		VERSION ${LIBVERSION}
```

Comment:
can you explain this? Your title mentions "soversion", but it remain unchanged on the line below. Instead you change the "version".

Reply:
I changed it in the variable on [top](https://github.com/arvidn/libtorrent/pull/4929/commits/b834883813fbd088073c3c978fc3a88a5d78dcf8#diff-af3b638bc2a3e6c650974192a53c7291R17).

Sorry about the rest, I've made a mess.
Will make a new PR when ready.